.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,237 @@
+        run: docker compose down --volumes --remove-orphans
+
+      - name: Push docker images
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 10
+          max_attempts: 3
+          command: |
+            docker compose push webknossos
+            docker compose push webknossos-datastore
+            docker compose push webknossos-tracingstore
+            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)